### ISSUE TITLE
Feature Request: votes left

### ISSUE THREAD

**Issue by davidak** [allourideas/allourideas.org#53]
As a user i would like to see how many votes are left until it is complete.

Sure this number grows exponentially with more pairs and can be demotivating, but it's stated clearly that you don't have to vote on all combinations.

Just a small text:

`votes left: 74`

**Comment by ockhamlabs** [comment]
Wondering if you resolved this somehow?

**Comment by msalganik** [comment]
I understand that urge, but the survey is actually never complete.  You can vote as much as you'd like or as little as you'd like.  The philosophy behind that choice is explained in our paper: http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0123483

**Comment by davidak** [comment]
@msalganik with for example 3 items, there are only 3 pairs. Do you see an option to calculate and show it or do you think it's better when the user don't know how many votes are left?

**Comment by msalganik** [comment]
True.  In most cases, however, there are large number of item; I generally recommend about 50, which makes 1225 pairs.  Most of our design decisions are focused on those type of wiki surveys.  I don't think we want to suggest to participants that they need to vote on everything.

**Comment by davidak** [comment]
>I don't think we want to suggest to participants that they need to vote on everything.

right, i never suggested that. but if a user **want** to do that, he might want to know if it takes him 10 minutes or 10 hours. i gave up after some time (felt like hours) and created this issue. maybe i'm not the regular user ;)

>1225 pairs

if a decision takes around 2 seconds, it would take 40 minutes. totally doable! (if you **really** want it)

maybe remind the user after each 15 minutes to take a break.
